### DISCUSSION TITLE
CIFS Share - Cannot Connect

### DISCUSSION THREAD

**Discussion by njperosion** [rern/rAudio#1825] [General]
Hello,

Feeling very silly as am an intermediate user and not had a problem beforehand - I'm unable to establish my NAS via CIFS. Setting up a Pi 3 + HiFiBerry Digi+ with raudio1 again as have a use case for it - previously been left by the wasteside since 2022 and an earlier raudio1 build.

IP address is correct, sharepoint is /volume1/music (Synology NAS), username and pass is as is for a special account I made just for music access. And all to no avail!

Tried NFS also (and added the special permissions - in fact they're still in place on my NAS from having raudio1 and even Runeaudio itself previously) and to no avail also. I enter the details, click okay, and some icon just pulses in the bottom left corner and no share is established.

Can I establish this share via ssh instead? Help!

**Comment by R101G** [comment]
Can you connect to the share from another device?

**Reply by njperosion** [reply]
TVs, VLC on a plethora of devices (Android phones, tablets, iPads, Quest 3), countless laptops and PCs. My entire home is oriented around the NAS - should have made it clear - apologies! 12-bay jet-engine Synology running so much stuff (data, apps, VMs) I can't even make a list. Working via 10G fibre and a couple of 1G links - only want to connect it via an IP for one of the 1G links which I use for other audio shares - Sonos notably (which has been fine for years). Got me stumped!

Not set much up on the raudio1 so may just try imaging it again from scratch, see if that has any luck. I do have a data folder from a previous raudio1 installation a couple years ago but just wanted to start a clean slate.

**Reply by R101G** [reply]
Just the NAS IP and share name is usually all you need. It could be that the NAS has an earlier Samba version, but that seems unlikely.

**Reply by njperosion** [reply]
Running DSM 7.2.1 and have SMB at minimum v2 and maximum v3. Nothing in the house struggles to access the storage.

About to rebuild the raudio1 again on the Pi and give it another whirl!

**Reply by R101G** [reply]
@njperosion 
rern has just posted an update which refers to adding network storage, so that might have been the problem.

**Reply by njperosion** [reply]
Yes it was - just ran the update and tried again - all successful! Didn't even need to declare /volume1/ either, just the straight name of the share.

Thanks for the heads up on that - I probably wouldn't have checked or assumed it to have been a bug.

**Comment by safefly** [comment]
Hi njperosion
I seem to be fighting the way you did and could do with a little help - my level is also intermediate...
I cannot make rAudio 20250215 access my Synology (DSM 6.2.4-25556 Update 8) music files
Any How To basic lead you could share ?
Thanks in advance
